pkgs/_analyzer_macros/lib/macro_implementation.dart

```diff
@@ -9,7 +9,7 @@ import 'package:analyzer/src/summary2/macro_declarations.dart' as analyzer;
 import 'package:analyzer/src/summary2/macro_injected_impl.dart' as injected;
 import 'package:dart_model/dart_model.dart';
 import 'package:macro_service/macro_service.dart';
-import 'package:macros/macros.dart' hide Code;
+import 'package:macros/macros.dart' as injected;
```

Comment:
I generally would prefer prefixes named after the package, so just `macros` in this case.

Reply:
The word `injected` I think in this case is also just super confusing to me? I think because the whole situation is so complicated, I don't really understand what is injected into what or how, and so this term is just confusing... at least for `macros` I know its the package:macros stuff?

Reply:
I also did not realize we were using overlapping prefixes, that is just even more confusing haha. I would call the other one `analyzer_macros` or something.

Reply:
Done.